LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2018 FURUHASHI Sadayuki
```

Comment:
Should this be `Copyright 2018 Fluentd Authors`?

Cc @edsiper

Reply:
What do you think of using `Copyright Fluentd Authors` instead?
(in accordance with the discussion we've been having for OpenTracing: https://github.com/opentracing/opentracing-go/pull/181#issuecomment-376668544)

Reply:
LGTM "Copyright Fluentd Authors" :)

Reply:
Should we mass-change the Copyright in the source code as well?

Reply:
@kzk Feel free, but it's not necessary.